client/locales/de.json

```diff
@@ -0,0 +1,402 @@
+        "zones": "Zonen",
+        "control": "Steuerung",
+        "settings": "Einstellungen",
+        "links": "Links",
+        "about": "Über"
+    },
+    "home": {
+        "labels": {
+            "area": "Bereich",
+            "time": "Zeit",
+            "battery": "Batterie",
+            "state": "Ladungszustand..."
+        },
+        "buttons": {
+            "start": "Start",
+            "pause": "Pause",
+            "stop": "Stop",
+            "home": "Home",
+            "spot": "Spot",
+            "find": "Finde",
+            "goto": "Gehe zu",
+            "zones": "Zonen",
+            "storeMap": "Karte speichern",
+            "loadMap": "Karte laden",
+            "unknownPower": "Unbekannte Leistung",
+            "invalid": "Ungültiger Button"
+        },
+        "popups": {
+            "selectFanspeed": "Auswahl des Leistungsmodus:",
+            "selectGotoTarget": "Setzte einen Zielpunkt:",
+            "selectZones": "Auswahl der Zonen zur Reinigung:",
+            "selectMapToLoad": "Auswahl des Speicherplatzes zum Laden der Karte:",
+            "emptyZoneChosen": "Es wurde eine leere Zone gewählt. Bitte zuerst die Zone einrichten.",
+            "tooManyZones": "Es wurde versucht {{count}} Unterzonen zu wählen. Es werden maximal 5 unterstützt.",
+            "chooseMapName": "Titel zum Speichern der Karte wählen:",
+            "invalidMapName": "Der Titel darf nur Buchstaben, Zahlen, Leerzeichen und Bindestriche enthalten.",
+            "slotAlreadyTaken": "Speicherplatz #{{slot}} ist bereits in Benutzung",
+            "nameAlreadyTaken": "Name \"{{name}}\" wurde schon verwendet",
+            "noMapsStored": "Keine gespeicherte Karten!",
+            "slotNo": "Speicherplatz #",
+            "confirmMapDelete": "Soll \"{{name}}\" wirklich gelöscht werden?",
+            "wait": "Warten",
+            "waitMapTicks": "Kartenersetzungsroutinen laufen. Bitte weitere {{seconds}} Sekunden warten...",
+            "waitMapFinished": "Beendet",
+            "waitMapEncourage": "Angeforderte Routinen sollten nun beendet sein.",
+            "mapStoredOK": "Karte erfolgreich gespeichert!",
+            "mapDeletedOK": "Gespeicherte Karte erfolgreich gelöscht!",
+            "mapRestoredOK": "Karte erfolgreich wiederhergestellt!"
+        },
+        "fanspeedPresets": {
+            "whisper": "Flüstern",
+            "quiet": "Leise",
+            "balanced": "Ausgeglichen",
+            "turbo": "Turbo",
+            "max": "Maximal",
+            "mop": "Wischen"
+        }
+    },
+    "map": {
+        "status": "Status",
+        "battery": "Batterie",
+        "connecting": "Verbinden...",
+        "statusText": "{{status, lowercase}}",
+        "errorText": "{{error, lowercase}}",
+        "confirmFullCleaning": "Zonenreinigung kann nicht gestartet werden: Keine Zonen spezifiziert. Soll stattdessen eine komplette Reinigung gestartet werden?",
+        "cannotResume": "Nichts zur Wiederaufnahme!",
```

Comment:
"cannotResume": "Keine Tätigkeiten zum Fortsetzen vorhanden!"

Reply:
Will change it to "Keine Tätigkeiten zur Fortsetzung!"